index.js

```diff
@@ -0,0 +1,131 @@
+$(document).ready(function() {
+  let loadTrips = function loadTrips() {
+    url = "https://trektravel.herokuapp.com/trips";
+    $.get(`${url}`, function(data) {
+      for (let trip of data) {
+        if (trip.continent != null) {
+          let tripInfo = $(
+            '<tr id="trip' + trip.id + '">'
+            + '<td class="trip-name">' + '<a class="trip-title" data-id =' + trip.id + '>' + trip.name + '</a>' + '</td>'
+            + '<td class="trip-cont">' + trip.continent + '</td>'
+            + '<td class="trip-week">' + trip.weeks + '</td>'
+            + '</tr>'
+          );
+          $('#trip-list').append(tripInfo);
+        }
+      }
+    });
+  };
+
+  let loadTrip = function loadTrip(id) {
+    url = `https://trektravel.herokuapp.com/trips/${id}`
+    $.get(`${url}`, function(data) {
+      data.category = data.category.charAt(0).toUpperCase() + data.category.slice(1);
+      data.cost = parseFloat(Math.round(data.cost * 100) / 100).toFixed(2);
+      let tripInfo = $(
+        '<tr class="trip-deets">'
+        + '<td colspan="3">'
+        + '<table class="deet-table">'
+        + '<tr>'
+        + '<td>' + '<i class="fa fa-tags"></i> Category: ' + data.category + '</td>'
+        + '<td>' + '<i class="fa fa-usd"></i> Cost: ' + data.cost + '</td>'
+        + '</tr>'
+        + '<tr>'
+        + '<td class="about-text" colspan="2">' + data.about + '</td>'
+        + '</tr>'
+        + '<tr>'
+        + '<td class="about-text" colspan="2"><button type="button" class="btn btn-warning reserve-go">Reserve Trip</button>' + '</td>'
+        + '</tr>'
+        + '</table>'
+        + '</td>'
+        + '</tr>'
+      );
+
+      $(`tr#trip${data.id}`).after(tripInfo);
+      $('div#reserve-modal').removeClass().addClass(`reserve-${data.id}`);
+    });
+  };
+
+  let details_exist = false;
+  let current_trip_id;
+
+  $('tbody#trip-list').on('click', 'a', function() {
+    let tripID = $(this).attr('data-id');
+
+    if (current_trip_id != tripID) {
+      $('tr[id^=trip]').css('background', 'none');
+      $('table.table').find('tr.trip-deets').each(function() {
+        $(this).remove();
+      });
+      details_exist = false;
+      current_trip_id = null;
+      loadTrip(tripID);
+      $(`tr#trip${tripID}`).css('background', '#777');
+      details_exist = true;
+      current_trip_id = tripID;
+    } else {
+      $('tr[id^=trip]').css('background', 'none');
+      $('table.table').find('tr.trip-deets').each(function() {
+        $(this).remove();
+      });
+      details_exist = false;
+      current_trip_id = null;
+    }
+  });
+
+  $('#all-trips').on('click', function() {
+    let imageHeight = $('section.hero-image').css('height');
+    if (imageHeight == '780px') {
+      $('section.hero-image').css('height', '100px');
+      $('div.image-wrapper').css('padding', '0');
+      $('div.title-row').css('height', '100%');
```

Comment:
I'm not a big fan of all this inline CSS.

A better way to organize things might be to have a couple of classes on the top-level element, something like `.intro-mode` or `.table-mode`, that you add and remove as appropriate. Then in your CSS, you could prefix your selectors with this if they apply to one mode or the other.

That's sort of a complex idea and I don't know if I've done a good job of explaining it, so see me if it's still unclear.